README.md

```diff
@@ -1,52 +1,167 @@
+
+## API Basics (done)
+
+We interact with web APIs through a set of urls. Each url defines a resource that we access.  The benefit of using a gem is that we don't have to build these urls ourselves.  The gems wrap the necessary code into neat little packages.  When we ask a gem for data, a gem will make a request to the API, handle the response and parse the data into a format we can use. Instead of relying on the magic of gems let's get our hands dirty.
+
+### Postman and the GitHub API (done)
+Let's see what's actually happening when a gem makes a request to an API. Ok, but how do we do this without a gem? We could write some ruby code, but let's try using a tool called Postman. Postman is an easy to use Chrome extension that lets us make different web requests.  It easily allows us to interact with web APIs. For our exercise, we are going to work with the GitHub API to retrieve information about the Ruby on Rails GitHub repository. To get started we need to setup Postman.
+
+#### Postman Installation (almost done)
+
+1. Visit https://www.getpostman.com and install the Chrome extension.
+2. Once you have it installed, open Postman from the Chrome Apps menu.
+3. Skip the signup and go straight to the app. 
+
+Great!  Now you're all set up to make your own API requests.  As you know, Rails is a large open source project with thousands of contributors.  Those contributors make a hefty amount of commits.  The GitHub API allows us to retrieve the list of commits made to the Rails repository.
+
+At this point Postman should be loaded and ready to go:
+
+* Enter **https://api.github.com/repos/rails/rails/commits** in the URL textbox.
+* Click the **Send** button.
+
+Once the request finishes, Postman will display the results.  Does this format look familiar?  If you said JSON then give yourself a pat on the back. What we're looking at is a JSON list of all the commits made to the Rails repository.  Each hash in the array has an author key.  Do you recognize any of the commiters?  If not, let's try to narrow the results returned from the GitHub API.
+
+GitHub exposes a way for us to do this using HTML parameters.  By changing the URL slightly to include the `author` parameter, we can ask the GitHub API to return only the commits made by DHH (bonus points if you know who this is!).
+
+Let's add ```?author=DHH``` to the end of the url and see how the results change.
+
+Go back to Postman and perform the following:
 
-## Intro
+* Enter **https://api.github.com/repos/rails/rails/commits?author=DHH**
+* Click the **Send** button.
 
-So far, when you've been using an API, you've been using a gem to access the API endpoints. Basically, you're relying on work someone else did to access clean data in the desired format.
 
-Instead of relying on a gem or library to do the heavy lifting for us, we can use Ajax to do it ourselves, without having a page refresh. 
+** Maybe link this back to rails or a gem example **
```

Comment:
I think the link back to a gem example could be really useful — maybe pointing to https://github.com/octokit/octokit.rb/blob/master/lib/octokit/client/commits.rb#L29 or similar to show how a gem builds up the URL just like the student is doing in Postman.